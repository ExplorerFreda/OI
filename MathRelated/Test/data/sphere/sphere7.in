9
1673.000000 845.500000 6455.000000 140.500000 3580.500000 19437.500000 19415.500000 3921.500000 7092.500000
12652.000000 19326.500000 17708.000000 4418.000000 7910.500000 2112.000000 1953.500000 10705.000000 3407.500000
11258.500000 3712.000000 11288.500000 11924.000000 3426.500000 18155.000000 14087.000000 11102.000000 7999.500000
83.500000 723.000000 3455.000000 9193.500000 18033.500000 16239.500000 10502.000000 7857.500000 3887.500000
7484.500000 19290.000000 3164.500000 19301.500000 7368.500000 19205.500000 14249.500000 1355.000000 12532.000000
16846.500000 3137.500000 11495.500000 17991.500000 19193.500000 10380.000000 17038.000000 13567.000000 8686.500000
4250.500000 14906.500000 3042.000000 6614.500000 1990.000000 6538.000000 19930.500000 17486.500000 14755.500000
5431.000000 17309.000000 7532.000000 13616.500000 6513.500000 7502.000000 4097.000000 7509.500000 7126.000000
6832.500000 7779.500000 11619.500000 15624.000000 17480.000000 17119.500000 17477.000000 16840.500000 319.500000
18523.500000 16828.500000 8066.500000 15293.500000 13300.000000 9281.500000 18943.000000 10139.500000 13058.000000
