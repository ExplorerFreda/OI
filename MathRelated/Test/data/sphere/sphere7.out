-20996.907 15626.169 54228.454 3911.545 29264.537 18119.481 14432.288 5030.087 50878.803
